survey/SHACL_SurveyResultDataOffering.ttl

```diff
@@ -0,0 +1,52 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix surveyonto: <http://semanticweb.org/metadatasurveyontology/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+surveyonto:InstantiatedVirtualResourceShape
+	a sh:NodeShape ; 
+	sh:targetClass surveyonto:InstantiatedVirtualResource ; 
```

Comment:
target class should point to the ontology class (which is currently missing)
You can have a look at the examples folder